### PR TITLE
Add new rule button

### PR DESCRIPTION
**Pull request by GivenTeacher5**
## Describe Your Changes
Added a button to create a new rule on click. 

## Type of Change
Please delete options that are not relevant.

- [x] New feature (change that adds functionality)

## Checklist (DOD)
- [x] The code is written and passes all Acceptance criteria's
- [x] Any Zombie code is removed
- [x] Code has proper comment documentation
- [x] The code is unit tested with happy case and passes
- [x] Code is refactored for optimization
- [x] The code has been peer-reviewed by one or more team members who did not work on it.
- [x] The code has no merge conflicts

### CODE REVIEW

File: front-end/src/app/app.component.html [hunk 1]
```diff
@@ -1,9 +1,9 @@
 <main class="d-flex flex-nowrap">
     <app-sidebar class="d-flex flex-column flex-shrink-0 p-3 text-bg-dark col-sm-3" style="width: 280px;"></app-sidebar>
-    <div class="col-sm-9">
+    <div class="col-sm-8">
```
Comment (ColeNelson806) [inline front-end/src/app/app.component.html:3]: Why was this switched to 8 from 9? Curious what the reasoning was.

**Review by ColeNelson806 — CHANGES_REQUESTED** [review]
After this edit, everything else looks good to me. Good test names and I am not seeing any obvious edge cases missing.

File: front-end/src/app/app.component.html (hunk 1 above)
Comment (GivenTeacher5) [reply front-end/src/app/app.component.html:3]: mistake

**Review by ColeNelson806 — APPROVED** [review]
Looks good
